swagger/public_v1/api_spec.json

```diff
@@ -1961,4 +1973,4 @@
       }
     }
   }
-}
+}
```

Comment:
Line ending?

Reply:
this is auto generated, i'm assuming the rake command will override any changes i make to this file